### PR TITLE
Polyhedron Demo: new batch of fixes

### LINKED ISSUE #5176: Boundary edges or split issue
I do:
"Detect boundaries", select the polyline item and then "Split Graph Into Polylines", then "split selected polyline" and got 2 polylines instead of one. There might be some duplicated edges that makes parallel edges to be inserted in a `boost::adajcency_list`.

Screenshot from 2020-11-20 14-08-36
[hex.off.gz](https://github.com/CGAL/cgal/files/5573647/hex.off.gz)

### PR DESCRIPTION
**Pull request by maxGimeno**
## Summary of Changes

- Fix the shading in scaled scenes.
- Add a new action to split polylines using a graph, but without the mesh_3 condition about angle.
- Fix crash when moving a group of polylines up or down
- Fix the scene that loads a ssh even when cancelled

## Release Management

* Affected package(s): Polyhedron demo
* Issue(s) solved (if any): fix #5176

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Plugins/IO/Polylines_io_plugin.cpp [hunk 1]
```diff
@@ -44,10 +46,16 @@ class Polyhedron_demo_polylines_io_plugin :
       actionSplit_polylines->setProperty("subMenuName", "Operations on Polylines");
       actionSplit_polylines->setObjectName("actionSplitPolylines");
 
+      actionSplit_polylines_graph= new QAction(tr("Split Graph Into Polylines"), mainWindow);
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Plugins/IO/Polylines_io_plugin.cpp:49]:
```suggestion
      actionSplit_polylines_graph= new QAction(tr("Split Graph into Polylines"), mainWindow);
```

File: Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Mesh_3_plugin.cpp [hunk 2]
```diff
@@ -101,7 +101,7 @@ class Mesh_3_plugin :
               this, SLOT(mesh_3_surface()));
     }
 
-    actionSplitPolylines = new QAction("Split polylines in a graph", mw);
+    actionSplitPolylines = new QAction("Split polylines in a graph (Tet Mesh)", mw);
```
Comment (sloriot) [inline Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Mesh_3_plugin.cpp:104]:
```suggestion
    actionSplitPolylines = new QAction("Split Polylines in a Graph (Tet Mesh)", mw);
```

Comment (sloriot) [reply Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Mesh_3_plugin.cpp:104]: @lrineau could you suggest a better name to Maxime? Build Feature Graph?

Comment (lrineau) [reply Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Mesh_3_plugin.cpp]: Yes, `Build Features Graph` is correct, maybe with `for Mesh_3` as a suffix.

**Comment by maxGimeno** [comment]
Successfully tested in https://cgal.geometryfactory.com/CGAL/testsuite/results-5.3-Ic-34.shtml

**Pull request merged**